crates/node/src/bench_utils.rs

```diff
@@ -803,25 +804,33 @@ impl ShieldedUtils for BenchShieldedUtils {
                 ContextSyncStatus::Speculative => SPECULATIVE_FILE_NAME,
             }
         };
-        let mut ctx_file = File::open(
-            self.context_dir.0.path().to_path_buf().join(file_name),
-        )?;
+        let Ok(mut ctx_file) =
+            File::open(self.context_dir.0.path().to_path_buf().join(file_name))
+        else {
+            // a missing file means there is nothing to load.
+            return Ok(());
+        };
```

Comment:
the exact error you'll get is [`NotFound`](https://doc.rust-lang.org/stable/std/io/enum.ErrorKind.html#variant.NotFound)
```suggestion
        let mut ctx_file =
            File::open(self.context_dir.0.path().to_path_buf().join(file_name)).or_else(|err| {
                if err.kind() == std::io::ErrorKind::NotFound {
                    Ok(())
                } else {
                    Err(err)
                }
            })?;
```